lapce-ui/src/tab.rs

```diff
@@ -1413,6 +1414,32 @@ impl LapceTab {
                             }
                         }
                     }
+                    LapceUICommand::SetLanguage(name) => {
+                        ctx.set_handled();
+                        let editor =
+                            if let Some(editor) = data.main_split.active_editor() {
+                                editor
+                            } else {
+                                return;
+                            };
+
+                        let mut doc = data.main_split.content_doc(&editor.content);
+                        let doc = Arc::make_mut(&mut doc);
+
+                        if name.is_empty() || name.to_lowercase().eq("plain text") {
+                            doc.set_syntax(None);
+                        } else {
+                            let lang =
+                                match LapceLanguage::from_name(name.to_string()) {
+                                    Some(v) => v,
+                                    None => return,
+                                };
+
+                            doc.set_language(lang);
+                        }
+                        doc.trigger_syntax_change(None);
+                        eprintln!("syntax changed: {:?}", doc.syntax());
```

Comment:
Something to clean up?

Reply:
Left it in case someone had issues but sure, can remove it